### PR TITLE
feat(ci): kubernetes deployment

### PR DESCRIPTION
**Pull request by themightychris**
Don't have this working yet, running into weird auth problems with my local kind cluster pulling from ghcr.io even though I had a `regcred` secret in place that looked good

This _might_ be enough for a working deploy after that, but I have no idea if my use of `hostAliases` will make the `app` and `db` hostnames hardcoded into the frontend nginx config and backend will work. Ultimately we want the backend connecting to a database provided through env, and to probably let k8s' ingress handle mounting `/api`

TODO:
- [ ] Remove line from Dockerfile
- [ ] Mount `/` with docker-compose
- [ ] Create ingress path for container on `/api`
- [ ] *Django App*: Receive DB Hostname and Creds through ENV

### CODE REVIEW

**Comment by themightychris** [comment]
ah ha, I've never used ghcr.io before, you have to explicitly switch a package to public after you publish it

**Comment by themightychris** [comment]
we should probably remove the line i added to dockerfile to copy nginx config in. If the container build without that provides for serving the static content out of /, then that can be something we just mount with docker-compose and the k8s ingress can handle plugging the api into the `/api` path on top of it

File: helm-chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
```
Comment (MooseQuest) [inline helm-chart/values.yaml:46]: Do we need to clean this up, or is just fine lingering here disabled?

Comment (MooseQuest) [reply helm-chart/values.yaml:46]: Will enable and add correct values via different branch

**Review by MooseQuest — APPROVED** [review]
Approving and Merging

**Pull request merged**